sdk/src/contexts/solana/context.ts

```diff
@@ -1239,9 +1250,6 @@ export class SolanaContext<
     console.info(`Setting Solana compute unit price to ${fee} microLamports`);
 
     return [
-      ComputeBudgetProgram.setComputeUnitLimit({
```

Comment:
I think setting the compute unit limit is unnecessary since, "If a SetComputeUnitLimit instruction isn’t provided, the limit is calculated as the product of the number of instructions in the transaction and the [default compute unit limit](https://github.com/solana-labs/solana/blob/4293f11cf13fc1e83f1baa2ca3bb2f8ea8f9a000/program-runtime/src/compute_budget.rs#L13)." - it could become an issue if the units we set aren't enough for the transaction (I think we should let the Solana runtime determine this value dynamically)

Reply:
makes sense 👍